PendingReleaseNotes

```diff
@@ -124,6 +124,14 @@
 * RBD: list-watchers C++ API (`Image::list_watchers`) now clears the passed
   `std::list` before potentially appending to it, aligning with the semantics
   of the corresponding C API (`rbd_watchers_list`).
+* Telemetry: Users who are opted-in to telemetry can also opt-in to
+  participating in a leaderboard in the telemetry public
+  dashboards (https://telemetry-public.ceph.com/). Users can now also add a
```

Comment:
Space after "dashboards"
```suggestion
  dashboards  (https://telemetry-public.ceph.com/). Users can now also add a
```

Reply:
@ljflores thanks for catching this, there was an issue with encoding